tests/test_cli_help_env.py

```
@@ -32,6 +32,14 @@ def test_help_topic_outputs_details(capsys):
     assert "Synchronize provider archives" in out
 
 
+def test_help_examples_flag(capsys):
```

Ruff (0.15.9)

[warning] Unused function argument: `capsys`

(ARG001)